site/src/components/BuildsTable/BuildsTable.tsx

```diff
@@ -37,72 +38,74 @@ export const BuildsTable: FC<BuildsTableProps> = ({ builds, className }) => {
   const styles = useStyles()
 
   return (
-    <Table className={className} data-testid="builds-table">
-      <TableHead>
-        <TableRow>
-          <TableCell width="20%">{Language.actionLabel}</TableCell>
-          <TableCell width="20%">{Language.durationLabel}</TableCell>
-          <TableCell width="40%">{Language.startedAtLabel}</TableCell>
-          <TableCell width="20%">{Language.statusLabel}</TableCell>
-          <TableCell></TableCell>
-        </TableRow>
-      </TableHead>
-      <TableBody>
-        {isLoading && <TableLoader />}
-        {builds &&
-          builds.map((build) => {
-            const status = getDisplayWorkspaceBuildStatus(theme, build)
-            const buildPageLink = `/@${username}/${workspaceName}/builds/${build.build_number}`
-
-            return (
-              <TableRow
-                hover
-                key={build.id}
-                data-testid={`build-${build.id}`}
-                tabIndex={0}
-                onKeyDown={(event) => {
-                  if (event.key === "Enter") {
-                    navigate(buildPageLink)
-                  }
-                }}
-                className={styles.clickableTableRow}
-              >
-                <TableCellLink to={buildPageLink}>{build.transition}</TableCellLink>
-                <TableCellLink to={buildPageLink}>
-                  <span style={{ color: theme.palette.text.secondary }}>
-                    {displayWorkspaceBuildDuration(build)}
-                  </span>
-                </TableCellLink>
-                <TableCellLink to={buildPageLink}>
-                  <span style={{ color: theme.palette.text.secondary }}>
-                    {new Date(build.created_at).toLocaleString()}
-                  </span>
-                </TableCellLink>
-                <TableCellLink to={buildPageLink}>
-                  <span style={{ color: status.color }} className={styles.status}>
-                    {status.status}
-                  </span>
-                </TableCellLink>
-                <TableCellLink to={buildPageLink}>
-                  <div className={styles.arrowCell}>
-                    <KeyboardArrowRight className={styles.arrowRight} />
-                  </div>
-                </TableCellLink>
-              </TableRow>
-            )
-          })}
-
-        {builds && builds.length === 0 && (
+    <TableContainer className={className}>
+      <Table data-testid="builds-table">
```

Comment:
Do we need a test id here or can we `findByRole` using `table`?

Reply:
I didn't know table is a role, lets try that.

Reply:
Ahh, but there are two tables on the page, the resources table and the builds-table.

Reply:
I don't mind if you want to leave it as-is, but for future reference, you can use an `aria-describedby` attribute on it, and then do something like `getByRole('table', {description: 'builds table'})`

Reply:
Awesome! I'm doing it.